src/python/pants/backend/project_info/dependees.py

```diff
@@ -27,7 +27,9 @@ class AddressToDependees:
 @rule
 async def map_addresses_to_dependees() -> AddressToDependees:
     # Get every target in the project so that we can iterate over them to find their dependencies.
-    all_explicit_targets = await Get(Targets, AddressSpecs([DescendantAddresses("")]))
+    all_explicit_targets = await Get(
+        Targets, AddressSpecs([DescendantAddresses("")], apply_target_filters=False)
```

Comment:
As mentioned offline, I think that it would be cleaner for the `Specs` object to be constructed with the relevant filters by the `SpecsCalculator`... afaict, no other instance will be re-executing the command line specs: they'll always be executing a new/different query, and the filters are intended for a particular query: the one from the CLI.